spaceship/lib/spaceship.rb

```diff
@@ -1,3 +1,4 @@
+require 'fastlane_core'
```

Comment:
@KrauseFx How about maintaining their own implementation so we have consistent behaviour in all the tools? 

@hjanuschka Otherwise, yeah, that's exactly what you'd need to do.